### ISSUE TITLE
attention mask and the pad token id were not set

### ISSUE THREAD

**Issue by sticktoFE** [DLYuanGod/TinyGPT-V#29]
**Describe the bug**
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:21017 for open-end generation.

**To Reproduce**
for stage
![1707188188584](https://github.com/DLYuanGod/TinyGPT-V/assets/2472797/47b434c6-c9f7-49cb-9535-8061abd99f77)
4 

**Expected behavior**
VQA

**Screenshots**
![1707188056654](https://github.com/DLYuanGod/TinyGPT-V/assets/2472797/23766c5d-37fe-41df-bc09-008b2e49d1f7)

**Desktop (please complete the following information):**
 - OS:wsl ubuntu22.04
 - Browser chrome,
